modules/core/src/adapters/deck-adapter.js

```diff
@@ -38,7 +39,6 @@ export default class DeckAdapter {
   constructor(sceneBuilder) {
```

Comment:
I think at the top of this class we set default values for properties and provide a type annotation. Can you do that for "enabled"?